src/js/pixi-examples.js

```diff
@@ -225,11 +225,14 @@ jQuery(document).ready(($) => {
 
             // Generate HTML and insert into iFrame
             let pixiUrl = '';
+            let pixiWebWorkerUrl = '';
 
             if (bpc.pixiVersionString === 'local') {
                 pixiUrl = 'dist/pixi.js';
+                pixiWebWorkerUrl = 'dist/webworker.js';
             } else { // other versions come from S3
                 pixiUrl = `https://d157l7jdn8e5sf.cloudfront.net/${bpc.pixiVersionString}/pixi-legacy.js`;
+                pixiWebWorkerUrl = '/* TODO */';
```

Comment:
For example: https://d157l7jdn8e5sf.cloudfront.net/dev/webworker.js

```suggestion
                pixiWebWorkerUrl = `https://d157l7jdn8e5sf.cloudfront.net/${bpc.pixiVersionString}/webworker.js`;
```

Reply:
Oops, I was too slow! You got it.